gatsby-config.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
       ETC: '0x1934945354BDe5b34F0c68E6AA78492050856D38',
     },
     facebookAppID: '2127644397453206',
```

Comment:
What FB app is this id referring to @crazko?

Reply:
It was created by @Ljuk67 so we could embed FB news feed on contact page (see https://github.com/ParalelnaPolisKE/paralelnapoliskosice.sk/issues/19). I later decided to remove it.